ansible/testbed-cli.sh

```diff
@@ -81,10 +81,11 @@ function read_file
  testbed_name=${line_arr[1]}
  topo=${line_arr[2]}
  ptf_imagename=${line_arr[3]}
- ptf_ip=${line_arr[4]}
- server=${line_arr[5]}
- vm_base=${line_arr[6]}
- dut=${line_arr[7]}
+ ptf=${line_arr[4]}
```

Comment:
Could you move `ptf` column to the place before comment to be in sync with TestbedProcessing.py
https://github.com/Azure/sonic-mgmt/blob/27f225f05f56cac2e9c63fba517425a521ef1160/ansible/TestbedProcessing.py#L234

Reply:
@msosyak Thanks for point out this. It seems that we have found this problem but we put ptf column in different places :).  I think it is better to put PTF attributes (hostname, IP and image) together. What do you think of this? I can modify TestbedProcessing.py if necessary.

Reply:
@baiwei0427: I agree with your reasoning. I think you should modify TestbedProcessing.py and any other files to keep the order consistent. @msosyak: Can you please provide the names of all other files which should have their order modified to match here?

Reply:
'ansible/TestbedProcessing.py' where we generate testbed.csv. 
In other places, testbed.csv is parsed by python and not sensitive for the order of column if I'm not missed something.